search/api.py

```diff
@@ -0,0 +1,261 @@
+"""
+Functions for search
+"""
+from itertools import islice
+import logging
+
+from django.conf import settings
+from django.contrib.auth.models import User
+from elasticsearch.helpers import bulk
+from elasticsearch.exceptions import NotFoundError
+from elasticsearch_dsl import Mapping
+from elasticsearch_dsl.connections import connections
+
+from profiles.models import Profile
+from profiles.serializers import ProfileSerializer
+from search.exceptions import ReindexException
+
+log = logging.getLogger(__name__)
```

Comment:
since this isn't an import, shouldn't this line have 2 blank lines preceding it and, if preferred, just 1 blank line succeeding it?